### PR TITLE
CLOUDP-194833: Drop support for ops manager 5

### PR DESCRIPTION
**Pull request by gssbzn**
CLOUDP-194833: OM5 is EOL we can stop testing against it

### CODE REVIEW

File: build/ci/evergreen.yml [hunk 1]
```diff
@@ -1093,111 +1093,6 @@ tasks:
       - func: "e2e test"
         vars:
           E2E_TAGS: om60,remote,sharded
-  # Deploy ops manager 5.0 and test against it
```
Comment (tibulca) [inline build/ci/evergreen.yml:1096]: 🎉

**Review by andreaangiolillo — APPROVED** [review]
👋 👋

**Review by tibulca — APPROVED** [review]

**Comment by andreaangiolillo** [comment]
Do we need to remove som e2e tests and evergreen tasks? https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/e2e/ops_manager/version_manifest_test.go#L15

**Comment by andreaangiolillo** [comment]
do we need to remove some e2e tags as well https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/e2e/ops_manager/version_manifest_test.go#L15?

**Comment by gssbzn** [comment]
> do we need to remove some e2e tags as well

we should, I'll do a follow up

**Pull request merged**